package.json

```diff
@@ -39,7 +32,7 @@
     "coveralls": "^2.11.9"
   },
   "dependencies": {
-    "meow": "^3.7.0"
+    "yargs": "^4.7.1"
```

Comment:
> I'm just curious why wouldn't you upgrade to the latest version?

We need to be on the same version as some other modules using yargs for an environment that can't use npm for module management.